test/ctia/entity/incident_test.clj

```diff
@@ -110,7 +110,7 @@
                 app
                 (assoc new-incident-maximal :title "foo")
                 :incidents
-                30
+                pagination-sample-size
```

Comment:
It's somewhat strage that changing a `30` to a `10` shaves 26s.

Obviously I don't oppose that in itself, but could we be masking some underlying issue?

(also, how many times have you measured things? Cloud runners tend to have inconsistent performance)

Reply:
Yes, most of the numbers don't make sense. They were all generated on cloud providers since that's where the problems are (no warmups, 2 cores, limited memory).

Even the numbers in the commits in this PR vary. Maybe this change does nothing. I think objectively it reduces the number of calls to ES, but that's about all I can say for sure.

Reply:
I added some words of caution to the description that these numbers should probably be ignored.

Reply:
Perhaps a better metric to minimize is the number of http calls or ES calls.